lib/model/folder_sendrecv.go

```diff
@@ -1681,7 +1681,7 @@ func (f *sendReceiveFolder) dbUpdaterRoutine(dbUpdateChan <-chan dbUpdateJob) {
 		// sync directories
 		for dir := range changedDirs {
 			delete(changedDirs, dir)
-			if !f.FolderConfiguration.DisableFsync {
+			if runtime.GOOS != "windows" && !f.FolderConfiguration.DisableFsync {
```

Comment:
I have come back to this commit, but I'm unable to see how to code should be changed here (or at least not with my limited coding skills). If the `changedDirs` map is built specifically for this loop, why is the `!f.FolderConfiguration.DisableFsync` condition applied here instead of before building the map?

What I mean is that I cannot see the difference between `if runtime.GOOS != "windows"` and `!f.FolderConfiguration.DisableFsync` when it comes to where each of them should be applied.

Reply:
This is a small simplification that both applies with this PR and without it. As in not required to be changed in this PR.